Content.Server/Antag/AntagSelectionSystem.cs

```diff
@@ -298,6 +302,24 @@ public void MakeAntag(Entity<AntagSelectionComponent> ent, ICommonSession? sessi
 
         if (session != null)
         {
+            var antagLoadout = LoadoutSystem.GetJobPrototype(def.PrefRoles.First());
+            var pref = (HumanoidCharacterProfile) _pref.GetPreferences(session.UserId).SelectedCharacter;
+
+            if (_prototypeManager.TryIndex(antagLoadout, out RoleLoadoutPrototype? roleProto))
```

Comment:
This looks like it's pulling out a random antag loadout and not the one that corresponds to the provided antagdefinition.

Reply:
It's getting the first prefRole since the preference roles are a list, not sure how to get the selected one

Reply:
changed it to use a loadout datafield instead